app/views/appointments/index.html.slim

```diff
@@ -8,11 +8,22 @@ table.table.table-bordered
       b Scheduled Date
     td
       b Room No
+    td
+      b Status
+    -if current_user.role == 'superadmin'
```

Comment:
since role is a  parameter of current_user, .superadmin? would return no method error.